README.rst

```diff
@@ -15,6 +15,9 @@ This repository consists of:
 * `torchtext.data <#data>`_: Generic data loaders, abstractions, and iterators for text (including vocabulary and word vectors)
 * `torchtext.datasets <#datasets>`_: Pre-built loaders for common NLP datasets
 
+Note: we are currently re-designing the torchtext library and make it more compatible with pytorch (e.g. ``torch.utils.data``). Several datasets have been written with the new abstract in `torchtext.experimental <https://github.com/pytorch/text/tree/master/torchtext/experimental>`_ folder. We also open an issue to discuss the new abstract, and users are welcome to leave feedback `here <https://github.com/pytorch/text/issues/664>`_. 
```

Comment:
nit: "[...] the torchtext library **to** make it more compatible [...]"
nit: "[...] have been written with the new **abstractions** in [...]"
nit: "[...] We also **created** an issue [...] discuss the new **abstraction**, and [...]"